### DISCUSSION TITLE
No sixel output from Canvas.print() on xterm

### DISCUSSION THREAD

**Discussion by jerch** [GuardKenzie/chafa.py#6] [Q&A] [answered]
Is there a way to force sixel output to output the sixel string?

Have a couple of terminals tested, that have working sixel support (eg. xterm), but chafa.py is only giving me an empty string as output.

**Comment by GuardKenzie** [comment, marked as answer]
Yes! xterm is one of the terminals not detected properly so instead of executing
```python
canvas.print()
```

you can execute
```python
canvas.print(fallback=True)
```
to supplement the auto-detected terminal sequences with fallback control sequences (in your case, sixel control sequences).

You just have to be careful when doing this since you will just get a long string of random letters and symbols if you try this on a terminal that doesn't support sixels (e.g. kitty).

 ### edit
Check out [this section in the tutorial](https://chafapy.mage.black/usage/tutorial.html#i-don-t-get-any-output-from-print) and [the docs for `Canvas.print()`](https://chafapy.mage.black/api/Canvas.html#chafa.Canvas.print)

**Reply by jerch** [reply]
Oh sorry, missed that section in the docs. Thx alot!
